setup.cfg

```diff
@@ -38,6 +38,62 @@ zip_safe = False
 package_dir = =src
 packages = find:
 
+[options.extras_require]
+docs =
+    alabaster~=0.7.12
```

Comment:
do we need to pin this?

I think that it can be removed as twisted is not using `alabaster` directly.